reward/addressbook_connector.go

```diff
@@ -43,26 +43,26 @@ var (
 	errAddressBookIncomplete = errors.New("incomplete node information from AddressBook")
 )
 
-type addressBookManager struct {
+type addressBookConnector struct {
 	bc                         *blockchain.BlockChain
-	governanceHelper           governanceHelper
+	gh                         governanceHelper
 	addressBookABI             string
 	addressBookContractAddress common.Address
```

Comment:
`addressBookABI` field in `addressBookConnector` sounds a bit redundant. 
How about using `ABI` and `ContractAddr`?

Reply:
reflect